packages/calcite-components/src/components/list-item/list-item.tsx

```diff
@@ -357,20 +378,32 @@ export class ListItem
     To account for this semantics change, the checks for (this.hasUpdated || value != defaultValue) was added in this method
     Please refactor your code to reduce the need for this check.
     Docs: https://qawebgis.esri.com/arcgis-components/?path=/docs/lumina-transition-from-stencil--docs#watching-for-property-changes */
+    if (changes.has("expanded")) {
+      this.open = this.expanded;
```

Comment:
I don't think this is correct. expanded should set expanded, open should set open.

I think what needs to happen is that open and expanded each need a custom setter to set the other one when one is set.